pom.xml

```diff
@@ -27,6 +27,7 @@
 		<maven-failsafe-plugin.version>3.0.0-M5</maven-failsafe-plugin.version>
 		<docker-maven-plugin.version>1.0.0</docker-maven-plugin.version>
 		<structured-logging.version>1.9.12</structured-logging.version>
+		<private-api-sdk-java.version>2.0.105</private-api-sdk-java.version>
```

Comment:
This version is out of date I'm afraid and doesn't have the latest changes to the `InternalCompanyInsolvency` object. `2.0.118` seems to be the [latest](https://github.com/companieshouse/private-api-sdk-java/releases/tag/2.0.118)

Reply:
We cannot use any other version other than 2.0.105, latest versions has warnings which makes the build fail.